docs/modules/ROOT/pages/getting-started/installation.adoc

```diff
@@ -0,0 +1,45 @@
+= Installation
+
+This document explains how to install and configure KubeArchive in your Kubernetes cluster.
+
+== Prerequisites
+
+* The `helm` CLI
+* A PostgreSQL instance
+* CertManager is installed on the Kubernetes cluster (+v1.9.1)
+* Knative Eventing is installed on the Kubernetes cluster (+v1.14.3)
+
+== Prepare the database
+
+1. Create a user and a password for KubeArchive to use.
+1. Create a database for the user to use.
+1. As the user, execute the `ddl.sql` we provide in the database, located at:
```

Comment:
Since you are providing the location right below this, providing the location in the text is redundant. Also, the file name needs to be tagged:

```suggestion
1. As the user, execute the [filename]`ddl.sql` we provide, located at:
```

Reply:
Using `[filename]` is on the Red Hat Asciidoc guide, but why it is used? Do you know? I rather not use it, if it does not add value.

Reply:
The square brackets used before a word in backticks are for role assignment, what in html means the `class` in the HTML element.

The specific docs are [here](https://docs.asciidoctor.org/asciidoc/latest/attributes/role/#assign-roles-to-formatted-inline-elements).

It is currently being rendered like that:
![image](https://github.com/user-attachments/assets/e4be0f0b-dda5-4515-8a81-ae56bb6fa701)

Now about the question of if we should keep adding this? It doesn't make any difference adding this class because there is no specific `CSS` rule for it but we may want to introduce it at some point. See how to do this in Antora [here](https://docs.antora.org/antora-ui-default/stylesheets/#add-a-new-css-rule).

Reply:
Ah, I see.